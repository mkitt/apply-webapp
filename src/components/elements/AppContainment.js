// @flow
import glamorous from 'glamorous'

export default glamorous.div({
  position: 'relative',
  maxWidth: 1392,
  marginRight: 'auto',
  marginLeft: 'auto',
  paddingRight: 16,
  paddingBottom: 16,
  paddingLeft: 16,
})
