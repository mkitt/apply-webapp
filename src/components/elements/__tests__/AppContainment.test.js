import React from 'react'
import renderer from 'react-test-renderer'
import AppContainment from '../AppContainment'

describe('<AppContainment />', () => {
  it('shows the correct snapshot tree for the <AppContainment> element', () => {
    const component = <AppContainment />
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('shows the correct snapshot tree for the <AppContainment> element with options', () => {
    const component = (
      <AppContainment my="auto" px={16} py={16} />
    )
    const tree = renderer.create(component).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
