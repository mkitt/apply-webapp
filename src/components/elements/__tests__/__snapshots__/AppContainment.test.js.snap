// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppContainment /> shows the correct snapshot tree for the <AppContainment> element 1`] = `
.css-0,
[data-css-0] {
  position: relative;
  max-width: 1392px;
  margin-right: auto;
  margin-left: auto;
  padding-right: 8px;
  padding-bottom: 16px;
  padding-left: 8px;
}

<div
  className="css-0"
/>
`;

exports[`<AppContainment /> shows the correct snapshot tree for the <AppContainment> element with options 1`] = `
.css-0,
[data-css-0] {
  position: relative;
  max-width: 1392px;
  margin-right: auto;
  margin-left: auto;
  padding-right: 16px;
  padding-bottom: 16px;
  padding-left: 16px;
  margin-top: auto;
  margin-bottom: auto;
  padding-top: 16px;
}

<div
  className="css-0"
/>
`;
